_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
When I tried running `npm run lint`, I got this message:
```
Cannot find module 'eslint-config-google

Referenced from: /Users/namchi/Desktop/electron/12-19/Current-Revisions/.eslintrc.js

Error: Cannot find module 'eslint-config-google'```

<img width="1213" alt="screen shot 2016-12-16 at 10 49 13 am" src="https://cloud.githubusercontent.com/assets/5871889/21274485/531fea42-c37d-11e6-99f9-1a52f10eef7c.png">